src/graph/DeleteVerticesExecutor.cpp

```diff
@@ -0,0 +1,152 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/DeleteVerticesExecutor.h"
+#include "storage/client/StorageClient.h"
+
+namespace nebula {
+namespace graph {
+
+DeleteVerticesExecutor::DeleteVerticesExecutor(Sentence *sentence,
```

Comment:
You'd better don't rename the cpp file,  i can't see the changes with before.  
You could use another pr just for rename operation.